app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :increment_post_likes_counter
```

Comment:
[OPTIONAL] Nice job making use of the rails active callbacks for your `comments`, `posts`, and, `likes` models, I suggest you make use of `after_save` rather than `after_create` for your callbacks 😉.

This is because even tho they are both callbacks that allow you to execute code after a record has been saved to the database, they are triggered at different points in the object's lifecycle😉.

The `after_create` callback is triggered after a new record has been created and saved to the database for the first time. It is specifically tied to the creation of a new record. example, when you want to perform some action that should only happen once, right after the initial creation of a new record. e.g., sending a welcome email to a user after they sign up👍🏼.

The `after_save` on the other hand is triggered after a record has been saved to the database, regardless of whether it is a new record or an existing one being updated. it is used when you want to perform an action every time a record is saved, whether it's during the initial creation or subsequent updates. This callback is more general and is not tied specifically to the creation process😉.

You can kindly read more about it to see what suits best👍🏼.